Resources/Prototypes/Loadouts/items.yml

```
@@ -756,15 +756,4 @@
-# - < DarkSide - LOST PARADISE
-
-- type: loadout
-  id: LPPLoadoutPaperIdCardUSSP
-  category: Items
-  cost: 3
-  items:
-    - LPPPaperIdCardUSSP
-
-# - > 
\ No newline at end of file
+    - DrinkVacuumFlask
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)